cpp/src/barretenberg/honk/composer/standard_honk_composer.test.cpp

```diff
@@ -303,6 +303,7 @@ TEST_F(StandardHonkComposerTests, VerificationKeyCreation)
         composer.create_add_gate({ a_idx, b_idx, c_idx, fr::one(), fr::one(), fr::neg_one(), fr::zero() });
         composer.create_add_gate({ d_idx, c_idx, a_idx, fr::one(), fr::neg_one(), fr::neg_one(), fr::zero() });
     }
+    composer.create_prover();
```

Comment:
why do you have to create a prover here now?

Reply:
This was previously done under the hood via lines like `if(!proving_key) { compute_proving_key(); }` in the verifier. I removed those lines so now this has to be done explicitly.